src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -872,6 +874,17 @@ export default class DefaultAudioVideoController
     return true;
   }
 
+  updateLocalVideoFromPolicy(): boolean {
+    // Update bandwidth without renegotiation
+    this.logger.info('Updating local video from policy without renegotiation');
+    if (this.meetingSessionContext.enableSimulcast) {
```

Comment:
Can you log whats going on in this function?  Also this always returns true, which is not accurate for when we are switching simulcast streams.  I think i accounted for this logic in my branch, maybe it was mistakenly removed when you stripped out the RTP header extension stuff.

Reply:
Fixed! Thanks!